### PR TITLE
PHPCS Proposal

### PR DESCRIPTION
**Pull request by rwrobe**
This is a proposed PHPCS implementation/ruleset, including a GH action to run composer install and test.

Please be aware that the standard versions are not frozen by default; After creating a project, you would need to freeze these.

See #411 for a demo of what this does when you run PHPCBF.

### CODE REVIEW

**Comment by rwrobe** [comment]
@JBCSU We need to add a GH oAuth key to the environment secrets with the key `GH_TOKEN`. Don't have perms to do it myself.

File: .github/workflows/phpcs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        uses: actions/cache@v1
+        with:
+          path: vendor
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/composer.lock') }}
+
+      - name: Install Dependencies
+        if: steps.cache-dependencies.outputs.cache-hit != 'true'
+        env:
+          WP_PLUGIN_ACF_KEY: ${{ secrets.WP_PLUGIN_ACF_KEY }}
+          WP_PLUGIN_GF_KEY: ${{ secrets.WP_PLUGIN_GF_KEY }}
```
Comment (jbrinley) [inline .github/workflows/phpcs.yml]: In addition to `WP_PLUGIN_GF_KEY`, we also need `WP_PLUGIN_GF_TOKEN` to use our GravityForms composer installer utility.

Comment (rwrobe) [reply .github/workflows/phpcs.yml]: Good call, I added this in ea2473de!

**Comment by rwrobe** [comment]
@jbrinley Composer versions have been updated. Not sure what happened there, but it should match master now, except for the PHPCS stuff in require-dev!

**Comment by jbrinley** [comment]
Subsumed into #479 and merged

**Pull request closed without merging**